packages/cli/src/commands/build.ts

```diff
@@ -486,17 +486,34 @@ export async function build({
           ['Metadata', metaUrl],
         ])
       );
-      console.log(
-        bold(`Publish ${bold(fullPackageRef)} to the registry and pin the IPFS data to ${filteredSettings.publishIpfsUrl}`)
-      );
-      console.log(`> ${`cannon publish ${fullPackageRef} --chain-id ${chainId}`}`);
+
+      const isMainPreset = preset === 'main';
+
+      if (isMainPreset) {
+        console.log(
+          bold(
+            `Publish ${bold(`${name}:${version}`)} to the registry and pin the IPFS data to ${
```

Comment:
And for this combination it can use `PackageReference.packageRef` which doesn't include the preset.